npm/test/clientfileprocessing.test.ts

```diff
@@ -215,13 +214,13 @@ test("progressCallback function does not update progress bar if percentage proce
     new S3UploadMock()
   )
 
-  fileProcessing.progressCallback(progressInformation)
+  fileProcessing.metadataProgressCallback(progressInformation)
 
-  checkExpectedPageState("") // this field is not populated because % > 50
+  checkNoPageStateChangeExpected()
 })
 
-test("progressCallback function does not change the HTML state if no progress bar present", () => {
-  setupUploadPageHTMLWithoutProgressBar()
+test("metadataProgressBarCallback function updates the progress bar with the percentage processed", () => {
```

Comment:
Name the function in the test description as the actual function name: "metadataProgressCallback"